circuit/views.py

```diff
@@ -1,7 +1,5 @@
-from django.http import HttpResponse
-from django.template import loader
+from django.views.generic import TemplateView
 
 
-def landingpage(request):
-    template = loader.get_template("landingpage.html")
-    return HttpResponse(template.render())
+class LandingPage(TemplateView):
```

Comment:
This should have been a separate Pull Request. Right now I have no chance to merge the rest without it.
But since it doesn't matter, if we use a function or class, it's in this case no problem.